### PR TITLE
Do not proceed when you encounter an error outside of tests

### PR DESCRIPTION
**Pull request by kwerle**
The change in Worker will keep the tests from passing.
This is enough to stop the bleeding.

### CODE REVIEW

**Comment by kwerle** [comment]
@jacobbloom Please take a look.

File: lib/rspec/multiprocess_runner/file_coordinator.rb [hunk 1]
```diff
@@ -69,7 +69,8 @@ def get_file
         else
           return nil # Malformed response, assume done, cease function
         end
-      rescue
+      rescue Exception => e
+        puts("Got exception #{e} in get_file")
```
Comment (jacobbloom) [inline lib/rspec/multiprocess_runner/file_coordinator.rb]: will this be sufficient info to know what line of what file caused it?

File: spec/features/exit_code_specs.rb [hunk 2]
```diff
@@ -26,6 +26,18 @@
     it { is_expected.to eq(1) }
   end
 
+  context 'on broken followed by success' do
```
Comment (jacobbloom) [inline spec/features/exit_code_specs.rb]: should have a spec that checks that we output what error occurred

File: lib/rspec/multiprocess_runner/file_coordinator.rb (hunk 1 above)
Comment (kwerle) [reply lib/rspec/multiprocess_runner/file_coordinator.rb]: Nope.  But the file assignment seems the most likely line to blow up.  Odds are good there won't be an identifiable file.

File: spec/features/exit_code_specs.rb (hunk 2 above)
Comment (kwerle) [reply spec/features/exit_code_specs.rb]: I poked at this, and the app doesn't really do that at the integration level.
`    Process.wait spawn(command, %i(out err) => '/dev/null')`

At some level, I kinda don't care.  Everything but the exit code is kind of messy logging stuff.  And this stuff is messier than most.  If it changes output/format/whatever, ain't nothing downstream gonna break.

I guess I could log into a file?  Or capture the process output?  Neither seems particularly elegant or useful.  What do you think?

**Pull request merged**